WcaOnRails/app/models/delegate_report.rb

```diff
@@ -6,6 +6,11 @@ class DelegateReport < ActiveRecord::Base
 
   attr_accessor :current_user
 
+  before_create :set_discussion_url
+  def set_discussion_url
+    self.discussion_url = "https://groups.google.com/forum/#!topicsearchin/wca-delegates/" + URI.encode(competition.name)
+  end
```

Comment:
Actually I've just relised that the best place for this is the [delegate_report method in the Competition model](https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/models/competition.rb#L587-L598).
(Or we can set the `equipment` template here, just be consistent)